exercises/practice/leap/src/leap.cbl

```diff
@@ -0,0 +1,11 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. LEAP.
+       ENVIRONMENT DIVISION.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       PROCEDURE DIVISION.
+       LEAP.
+	  * Enter solution here
+		 CONTINUE.
```

Comment:
@axtens, these two lines contain tabs. Can that lead to problems?

Reply:
Fixed